src/test/scala/mesosphere/marathon/core/instance/TestInstanceBuilder.scala

```diff
@@ -163,6 +167,31 @@ object TestInstanceBuilder {
     new Instance(task.taskId.instanceId, Some(agentInfo), state, tasksMap, task.runSpecVersion, unreachableStrategy, None)
   }
 
+  def runningInstance(appId: PathId, version: Timestamp, instanceTracker: InstanceTracker): Future[Instance] = async {
```

Comment:
I much rather avoid this method. If there is some repetition in test we should have helper methods there. See my comment above.